src/components/Button.tsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react'
+
+export interface ButtonInterface {
+  handler: () => void
+  height: string
+  width: string

```

Comment:
If height and width aren't used in the component are they needed in the interface?

Reply:
No they're not, forgot to remove them